engine/entity.shader

```diff
@@ -0,0 +1,58 @@
+shader_type canvas_item;
+render_mode unshaded;
+
+uniform bool is_hurt = false;
+uniform bool custom_colors = false;
+
+uniform vec4 skin : hint_color = vec4(0.969, 0.71, 0.545, 1.0);
+uniform vec4 outline : hint_color = vec4(0.0, 0.0, 0.0, 1.0);
+uniform vec4 accent : hint_color = vec4(0.094, 0.522, 0.969, 1.0);
+
+uniform vec4 hurt_skin : hint_color = vec4(0.847, 0.0, 0.0, 1.0);
```

Comment:
please change to `uniform vec4 hurt_skin : hint_color = vec4(0.0, 0.0, 0.0, 1.0);` then we good